### PR TITLE
Update package-management.md

### PR DESCRIPTION
**Pull request by ghosted-sound**
Corrected grammar; Added bullets, punctuation & white space for readability.

### CODE REVIEW

File: docs/csharp/package-management.md [hunk 1]
```diff
@@ -33,7 +43,7 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
 
     ![Screenshot showing command palette search bar with placeholder text that reads "Enter a search term to search for a NuGet package."](images/package-management/nuget-command-addpackage-search1.png)
 
-    The Quick Pick shows a list of example NuGet packages to choose from.
+   The Quick Pick shows a list of example NuGet packages to choose from.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:46]:
```suggestion
    The Quick Pick shows a list of example NuGet packages to choose from.
```

Comment (ntrogh) [reply docs/csharp/package-management.md:46]: Keep indentation aligned with the screenshots.

File: docs/csharp/package-management.md [hunk 2]
```diff
@@ -49,13 +59,13 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
 
 2. If you have more than one project in your solution, you are asked to select which project contains the package that you would like to update.
 
-3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you see a message notifying you of this.
+3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you will see a message notifying you of this.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:62]:
```suggestion
3. Next, choose which package to update from the list of packages that are currently installed in your project and that have an update available. If no packages in your solution have an update available, you see a message notifying you of this.
```

File: docs/csharp/package-management.md [hunk 3]
```diff
@@ -49,13 +59,13 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
...
 
 4. Finally, you can select which version you would like to update the package to from a dropdown list of available versions.
 
     ![Screenshot showing quickpicks dropdown menu with placeholder text that reads: "Select a NuGet package version". The quickpick options show a list of example NuGet packages to choose from. There are indicators on the list to show which version is currently installed in the users project ("current"), and which is the latest available version ("latest")](images/package-management/nuget-command-update-versionselector.png)
 
-5. C# Dev Kit then performs the necessary changes and updates your project file and references.
+5. The C# Dev Kit then performs the necessary changes and updates to your project file and references.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:68]:
```suggestion
5. C# Dev Kit then performs the necessary changes and updates your project file and references.
```

Comment (ntrogh) [reply docs/csharp/package-management.md:68]: We don't use an article before the official name C# Dev Kit

File: docs/csharp/package-management.md [hunk 4]
```diff
@@ -65,20 +75,22 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
 
 3. Next, choose which package you want to remove from the list of packages that are currently installed in your project.
 
-4. C# Dev Kit then performs the necessary changes and updates your project file and references.
+4. The C# Dev Kit then performs the necessary changes and updates your project file and references.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:78]:
```suggestion
4. C# Dev Kit then performs the necessary changes and updates your project file and references.
```

File: docs/csharp/package-management.md [hunk 5]
```diff
@@ -10,14 +10,24 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
 
 * [NuGet Commands in C# Dev Kit](#nuget-commands-in-c-dev-kit)
 * [Dependency Management](#dependency-management)
 
+To learn more about NuGet, go to the [NuGet documentation](https://learn.microsoft.com/nuget/what-is-nuget).  
+
 ## NuGet commands in C# Dev Kit
 
-In C# Dev Kit, you can perform NuGet package operations in either of two ways: by using the Command Palette (`kb(workbench.action.showCommands)`) or by right-clicking in the C# Dev Kit Solution explorer. The following sections describe how to use each NuGet command through both of the two methods.
+In the C# Dev Kit, you can perform NuGet package operations in one of two ways:
```
Comment (ntrogh) [inline docs/csharp/package-management.md:24]:
```suggestion
In C# Dev Kit, you can perform NuGet package operations in one of two ways: 
```

File: docs/csharp/package-management.md [hunk 6]
```diff
@@ -10,14 +10,24 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
 * [Dependency Management](#dependency-management)
 
+To learn more about NuGet, go to the [NuGet documentation](https://learn.microsoft.com/nuget/what-is-nuget).  
+
 ## NuGet commands in C# Dev Kit
 
-In C# Dev Kit, you can perform NuGet package operations in either of two ways: by using the Command Palette (`kb(workbench.action.showCommands)`) or by right-clicking in the C# Dev Kit Solution explorer. The following sections describe how to use each NuGet command through both of the two methods.
+In the C# Dev Kit, you can perform NuGet package operations in one of two ways: 
+
+* Using the Command Palette (`kb(workbench.action.showCommands)`).
```
Comment (ntrogh) [inline docs/csharp/package-management.md:26]:
```suggestion
* Using the Command Palette (`kb(workbench.action.showCommands)`)
```

File: docs/csharp/package-management.md [hunk 7]
```diff
@@ -10,14 +10,24 @@ MetaDescription: C# package management with NuGet in Visual Studio Code
...
+
+* Right-clicking in the C# Dev Kit Solution explorer.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:28]:
```suggestion
* Right-clicking in the C# Dev Kit Solution explorer
```

File: docs/csharp/package-management.md [hunk 8]
```diff
@@ -65,20 +75,22 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
...
 
-4. C# Dev Kit then performs the necessary changes and updates your project file and references.
+4. The C# Dev Kit then performs the necessary changes and updates your project file and references.
 
 ## Dependency management
 
 ### Automatic NuGet restore
 
-For a .NET project, a package restore happens automatically when you create a project from a template, build, load, or make changes to an SDK-style project. You can view in the progress and logs in the Ouput panel.
+For a .NET project, a package restore happens automatically when you create a project from a template, build, load, or when making changes to an SDK-style project. You can view in the progress and logs in the Ouput panel.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:84]:
```suggestion
For a .NET project, a package restore happens automatically when you create a project from a template, build, load, or make changes to an SDK-style project. You can view in the progress and logs in the Ouput panel.
```

File: docs/csharp/package-management.md [hunk 9]
```diff
@@ -65,20 +75,22 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
...
 
 ![Package references in the Solution Explorer](images/package-management/package-references-solution-explorer.png)
 
-Packages that don't install properly when a restore happens or you run a build will show error icons in **Solution Explorer**.
+Packages that don't install properly when a restore happens or when you run a build, will show error icons in **Solution Explorer**.
+
+**Note**: At this time, you cannot right-click on projects to manage your NuGet packages and there is not a NuGet Package Manager user interface in Visual Studio Code. 
 
-**Note**: At this time, you cannot right-click on projects to manage your NuGet packages and there is not a NuGet Package Manager user interface in Visual Studio Code. For more information on managing packages, see [Install and manage NuGet packages with the dotnet CLI](https://learn.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli).
+For more information on managing packages, see [Install and manage NuGet packages with the dotnet CLI](https://learn.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli).
```
Comment (ntrogh) [inline docs/csharp/package-management.md:96]: The link to docs is related to the note and should be included in the blockquote. Please revert this change.

**Review by ntrogh — CHANGES_REQUESTED** [review]
@ghosted-sound Thanks for helping improve our docs. Please apply the requested changes.

File: docs/csharp/package-management.md (hunk 2 above)
Comment (ghosted-sound) [reply docs/csharp/package-management.md:62]: RE: you will see vs. you see

"You see" denotes it is appearing right now.

"You will see " is better grammar & makes more sense from a user standpoint

File: docs/csharp/package-management.md (hunk 9 above)
Comment (ghosted-sound) [reply docs/csharp/package-management.md:96]: ok, will do.

File: docs/csharp/package-management.md (hunk 2 above)
Comment (ntrogh) [reply docs/csharp/package-management.md:62]: In this context (product docs), this phrase refers to a _real condition_, where every time the condition is met, the outcome is always true. In technical docs we want to be as much "in the moment" as possible.

File: docs/csharp/package-management.md (hunk 9 above)
Comment (ghosted-sound) [reply docs/csharp/package-management.md:96]: Done.

File: docs/csharp/package-management.md [hunk 10]
```diff
@@ -65,7 +75,7 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
 
 3. Next, choose which package you want to remove from the list of packages that are currently installed in your project.
 
-4. C# Dev Kit then performs the necessary changes and updates your project file and references.
+4. The C# Dev Kit then performs the necessary changes and updates your project file and references.
```
Comment (ntrogh) [inline docs/csharp/package-management.md:78]:
```suggestion
4. C# Dev Kit then performs the necessary changes and updates your project file and references.
```

**Review by ntrogh — APPROVED** [review]

**Review by bpasero — APPROVED** [review]
